### PR TITLE
Migration of test_peer_status

### PR DESCRIPTION
**Pull request by nishith-vihar**
Migration of [test_case](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_peer_status.py) from glusto to redant

Updates: #292
Fixes: #552
Signed-off-by: Nishith Vihar Sakinala <nsakinal@redhat.com>

### CODE REVIEW

File: tests/functional/glusterd/test_peer_status.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        redant.setup_volume(self.vol_name, self.server_list[0],
+                            self.vol_type_inf[self.conv_dict['dist']],
+                            self.server_list[0:2], self.brick_roots,
+                            True)
+
+        # peer probe to a new node, N3
+        redant.peer_probe(self.server_list[2], self.server_list[0])
+
+        # Validate first three peers are in connected state
+        # In jenkins The next step which is add-brick from third node
```
Comment (srijan-sivakumar) [inline tests/functional/glusterd/test_peer_status.py:86]: nit..we can remove the jenkins related comment out.

**Comment by nishith-vihar** [comment]
```
test_peer_status-Generic PASS

Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   0   |      0       |
|   Disruptive  |   1   |    100.0     |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 test_peer_status
 
+-------------+-------------+-------------------+
| Volume Type | Test Result |  Time taken (sec) |
+-------------+-------------+-------------------+
|   Generic   |     PASS    | 48.00857067108154 |
+-------------+-------------+-------------------+



Framework runtime : 50.268097162246704
```

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: tests/functional/glusterd/test_peer_status.py (hunk 1 above)
Comment (nishith-vihar) [reply tests/functional/glusterd/test_peer_status.py:86]: Ack

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Pull request merged**